sdk/src/main/java/io/radar/sdk/util/RadarHandler.kt

```diff
@@ -0,0 +1,17 @@
+package io.radar.sdk.util
+
+import android.os.Handler
+
+/**
+ * Simple wrapper for passing a [Handler] as a [RadarPostable]
+ */
+internal class RadarHandler(private val handler: Handler) : RadarPostable {
```

Comment:
I'm not sure I understand what this and the `RadarPostable` do. `RadarPostable` is an `interface`, and `RadarHandler` subclasses `Handler` to implement `RadarPostable`?

Reply:
This is meant to abstract away from Android's `Handler` methods with are (generally) used to "post" executable code to the main thread. This behavior can make tests unpredictable, so by creating this wrapper object, we can easily mock this functionality and make it entirely synchronous in tests. I prefer abstracting away from Android objects so that we can mock that behavior, since unit tests are not meant to verify the behavior of the Android APIs.